Jenkinsfile

```diff
@@ -50,7 +50,7 @@ pipeline {
 						}
 					} catch(Exception e) {
 						readFile('DeployList.csv').split('\n').eachWithIndex { line, index ->
-							if (index < countCompleted) { 
+							if (index <= countCompleted) { 
```

Comment:
@msimp019 
I want to call attention to this line. I think i makes sense in terms of the deployment scripts, but am wondering if it is going to break the Ensemble Deploy/Rollback functionality if we are trying to rollback a non deployed package.

Reply:
per discussion, will set autorevert to false and call rollback on the failed deployment as well as the successful deployments in order to have the opportunity to call rollback scripts